sov-modules/sov-modules-macros/tests/dispatch/modules.rs

```diff
@@ -54,20 +59,25 @@ pub mod second_test_module {
     pub struct TestType {}
 
     #[derive(ModuleInfo)]
-    pub(crate) struct SecondTestStruct<C: Context> {
+    pub struct SecondTestStruct<K: Context> {
```

Comment:
Nit: Using `K` for `Context` is a little bit confusing. Could we use `Ctx` instead?

Reply:
fixed